### ISSUE TITLE
Store protocols tokens locked up

### ISSUE THREAD

**Issue by fabioDMFerreira** [DefiLlama/defillama-server#1]
This is a feature request.

I identified some steps to accomplish the request:

1- Changing computeTVL in @defillama/sdk to return the tokens locked up. Here is one suggestion of the output format for REN protocol
```
{
  total: 1021225134.1998382,
  tokens: [
    {
      usdAmount: 219793000,
      tokenSymbol: 'REN',
      amount: 184700000,
      price: 1.19
    },
    {
      usdAmount: 801432134.1998382,
      tokenSymbol: 'USDT',
      amount: 801661409.362916,
      price: 0.999714
    }
  ]
}
```
2- Adapting `storeNewTvl` to store the protocol's tokens TVL in dynamo DB (https://github.com/DefiLlama/defillama-server/blob/master/src/utils/storeNewTvl.ts)

**Comment by 0xngmi** [comment]
Yeah, this is something we are working on. Feel free to drop a PR implementing it!

**Comment by 0xngmi** [comment]
Implemented on #1, I think this could be closed?
